ui/js/controllers/perf/alerts.js

```diff
@@ -171,7 +174,81 @@ perf.controller('AlertsCtrl', [
+                                                      {selected: true});
+                        var testDescriptions = _.uniq(_.map(selectedAlerts, function(alert) {
+                            var suitekey = alert.series_signature.suite;
+                            var testkey = alert.series_signature.suite + '_' +
+                                alert.series_signature.test;
+                            var prefix = 'https://wiki.mozilla.org/Buildbot/Talos/Tests#';
+                            if (phTalosDocumentationMap[suitekey]) {
+                                return prefix + phTalosDocumentationMap[suitekey];
+                            } else if (phTalosDocumentationMap[testkey]) {
+                                return prefix + phTalosDocumentationMap[testkey];
+                            } else {
+                                // assume testkey will work otherwise
+                                return prefix + testkey;
+                            }
+                        }));
+                        var talosSuites = _.uniq(_.map(selectedAlerts, function(alert) {
+                            return alert.series_signature.suite;
+                        }));
+                        var trySuites = _.uniq(_.map(selectedAlerts, function(alert) {
+                            var suiteName = trySuiteMapping[alert.series_signature.suite];
+                            if (_.contains(alert.series_signature.test_options, 'e10s')) {
+                                suiteName += '-e10s';
+                            }
+                            return suiteName;
+                        }));
+
+                        // they have 3 days from today to respond (assuming
+                        // that holidays/weekends are not a reason to avoid
+                        // a backout, since the whole point is to be able to
+                        // do one before a pile of patches land on top of it)
+                        var backoutDate = new Date(Date.now() + 3*86400*1000);
+                        var dayMapping = {
+                            0: 'Sunday',
+                            1: 'Monday',
+                            2: 'Tuesday',
+                            3: 'Wednesday',
+                            4: 'Thursday',
+                            5: 'Friday',
+                            6: 'Saturday'
+                        };
+                        var compiled = $interpolate(template)({
+                            revision: alertSummary.resultSetMetadata.revision,
+                            alertHref: thServiceDomain + '/perf.html#/alerts?id=' +
+                                alertSummary.id,
+                            testDescriptions: testDescriptions.join('\n'),
+                            trySuites: trySuites.join(','),
+                            talosTestListSyntax: talosSuites.join(":"),
+                            backoutDay: dayMapping[backoutDate.getDay()]
```

Comment:
test descriptions - this maps to the wiki, but we .join() together, I assume this will be a list of full http:// urls.
backoutDay- how does this code ignore sat/sun?